web-ui/src/main/resources/catalog/components/viewer/wps/WpsService.js

```diff
@@ -240,14 +240,27 @@
           // when bbox is cleared value is still set to ',,,'
           if (input.boundingBoxData && data && data != ",,,") {
             var bbox = data.split(",");
+            
+            var geomCrs = "EPSG:4326";
+            if (input.boundingBoxData._default &&
+              input.boundingBoxData._default.crs &&
+              input.boundingBoxData._default.crs !== geomCrs) {
+              geomCrs = input.boundingBoxData._default.crs;
+              try {
+                bbox = ol.proj.transformExtent(bbox, "EPSG:4326", geomCrs);
+              } catch (e) {
+                console.warn("WPS | Failed to convert boundingBoxData to requested CRS " + geomCrs);
```

Comment:
If fails, the code continues to call the WPS service, is that expected?

Reply:
Yes for now. We don't have that many example of WPS using custom projection. Probably something to take care of at some point. Also WPS may return an error message in this case.